cypress/e2e/course/shareAndImportCourseWithDifferentSchoolOnStaging.feature

```diff
@@ -1,16 +1,18 @@
+# NOTE: This feature should only be executed in the staging environment due to the school API limitation,
+# which prevents creating two separate schools in the same scenario and using the copied URL from Scenario One
+# in Scenario Two due to new sessions.
+
 @regression_test
 @stable_test
-@group_V
-
-# Note: This feature should only be executed in the staging environment due to the school API limitation, which prevents creating two separate schools in the same scenario and using the copied URL from Scenario One in Scenario Two due to new sessions.
-
-Feature: Course - Teacher shares a course to other teacher from different school
+@group-F
+@prio_0_staging
```

Comment:
for an example, here using @prio_0_staging tag is correct because this feature is only for staging.